metatheory/metatheory.cabal

```diff
@@ -0,0 +1,1037 @@
+cabal-version:       2.4
+name:                plc-agda
+version:             0.1.0.0
+synopsis:            Command line tool for running plutus core programs
+homepage:            https://github.com/input-output-hk/plutus
+license: Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+author:              James Chapman
+maintainer:          james.chapman@iohk.io
+category:            Development
+extra-source-files:  README.md
+
+executable plc-agda
+  default-language:  Haskell2010
+  main-is:           Main.hs
+  other-modules:
+    Raw
```

Comment:
While you're touching this... it would make this file a lot easier to work with if there was a library which had all the generated modules and then the executable and the tests just depended on the library. Then you wouldn't have to duplicate the list of generated modules three times!